Doc/library/sqlite3.rst

```diff
@@ -424,6 +424,40 @@ Module functions
    Note: *typename* and the name of the type in your query are matched
    case-insensitively.
 
+.. function:: Date(year, month, day)
+
+   This function constructs an object holding a date value.
+
+.. function:: Time(hour, minute, second)
+
+   This function constructs an object holding a time value.
+
+.. function:: Timestamp(year, month, day, hour, minute, second)
```

Comment:
Are the second allowed to be fractional or not? (same question for `Time()`)

Reply:
In my tests, Doing something like this fails:

```python
sqlite3.Time(12, 30, 45.12)
```

But this succeeds:

```python
sqlite3.Time(12, 30, 45, 12)
```

However, it is not mentioned in PEP-249. How should we handle that?

Reply:
Oh. I think should definitely be documented and the signature of `Time` should be something like `Time(h, m, s[, ms])` (check the syntax in the time module, I don't remember now).

Reply:
Gotcha. Re-reading the issue, I believe the intention was to document these constructors per PEP-249. I'm not sure if this is more an implementation detail. Will wait for @erlend-aasland guidance.